qcelemental/tests/test_importing.py

```diff
@@ -39,6 +39,50 @@ def test_which_import_f_raisemsg():
     assert str(e.value).endswith("Python module 'evilpint' not found in envvar PYTHONPATH. Install `evilpint`.")
 
 
+def test_which_import_t_submodule():
+    ans = qcel.util.which_import('pint.util')
+    assert ans.split(os.path.sep)[-1] == 'util.py'
+
+
+def test_which_import_t_submodule_altsyntax():
+    ans = qcel.util.which_import('.util', package='pint')
+    assert ans.split(os.path.sep)[-1] == 'util.py'
```

Comment:
I would have thought that only this case (of the three positive) ones was needed. If we want to permit `which_import('pint.util')`, can take the arg and `module.rsplit('.', 1)` it and send the `[0]` to `package=`.

Reply:
It turns out that sending to the `package` kwarg actually doesn't change 'importlib.util.find_spec`'s behavior as we (I) had thought. Apologies for the mistake on my part. It turns out that there really isn't a way around a try-catch here (see my comment below).